.travis.yml

```diff
@@ -1,34 +1,49 @@
-language: python
-python:
-  - '2.7'
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+dist: bionic
+stages:
+  - test
 
-env:
-  global:
-    - BS_PIP_ALLOWED=1
-    - BS_ECHO_DEBUG=1
-    - SALT_ARGS="-l debug --local --retcode-passthrough --pillar-root=test/pillar --file-root=$PWD"
-  matrix:
-    - STATE=rsyncd
-    - STATE=rsyncd.config
+sudo: required
+cache: bundler
+language: ruby
 
-before_install:
-  - sudo apt-get update
-  - curl -L http://bootstrap.saltstack.org | sudo -E sh -s -- stable
+services:
+  - docker
 
-install:
-  # See what kind of travis box you're on to help with making your states
-  # compatible with travis
-  - sudo salt-call grains.items --local
+# Make sure the instances listed below match up with
+# the `platforms` defined in `kitchen.yml`
+env:
+  matrix:
+    - INSTANCE: default-debian-10-develop-py3
+    - INSTANCE: default-ubuntu-1804-develop-py3
+    - INSTANCE: default-centos-7-develop-py3
+    - INSTANCE: default-fedora-30-develop-py3
+    - INSTANCE: default-opensuse-leap-15-develop-py3
+    - INSTANCE: default-amazonlinux-2-develop-py2
+    - INSTANCE: default-arch-base-latest-develop-py2
+    - INSTANCE: default-debian-9-2019-2-py3
+    - INSTANCE: default-ubuntu-1804-2019-2-py3
+    - INSTANCE: default-centos-7-2019-2-py3
+    - INSTANCE: default-fedora-30-2019-2-py3
+    - INSTANCE: default-opensuse-leap-15-2019-2-py3
+    - INSTANCE: default-amazonlinux-2-2019-2-py2
+    - INSTANCE: default-arch-base-latest-2019-2-py2
+    - INSTANCE: default-debian-9-2018-3-py2
+    - INSTANCE: default-ubuntu-1604-2018-3-py2
+    - INSTANCE: default-centos-7-2018-3-py2
+    - INSTANCE: default-fedora-29-2018-3-py2
+    - INSTANCE: default-opensuse-leap-15-2018-3-py2
+    - INSTANCE: default-amazonlinux-2-2018-3-py2
+    - INSTANCE: default-arch-base-latest-2018-3-py2
+    - INSTANCE: default-debian-8-2017-7-py2
+    - INSTANCE: default-ubuntu-1604-2017-7-py2
+    - INSTANCE: default-centos-6-2017-7-py2
+    - INSTANCE: default-fedora-29-2017-7-py2
+    - INSTANCE: default-opensuse-leap-15-2017-7-py2
+    - INSTANCE: default-amazonlinux-2-2017-7-py2
+    - INSTANCE: default-arch-base-latest-2017-7-py2
```

Comment:
Since `fedora` isn't working, please use the following matrix:

```yaml
        platforms_matrix:
          # [os           , os_ver, salt_ver, py_ver,    inspec_suite]
          - [debian       ,  10   ,  develop,      3,         default]
          - [ubuntu       ,  18.04,   2019.2,      3,         default]
          - [amazonlinux  ,   2   ,   2019.2,      2,         default]
          - [arch-base    , latest,   2019.2,      2,         default]
          # - [fedora       ,  29   ,   2018.3,      2,         default]
          - [opensuse/leap,  15   ,   2018.3,      2,         default]
          - [centos       ,   6   ,   2017.7,      2,         default]
```